packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -61,6 +61,12 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
      */
     private _isBoundToContext: boolean = false;
 
+    private _referencedRoutes: string[] = [];
```

Comment:
It feels like it should be a set, not an array

Reply:
Yeah, set makes more sense. Eventually GC uses set so it will filter duplicates but its a good idea to do it at this level too.